draft-ietf-rats-architecture.md

```diff
@@ -679,13 +679,19 @@ In the background-check model, this Evidence may also be revealed to Relying Par
 
 ## Evidence
 
-Today, Evidence tends to be highly device-specific, since the information in the Evidence
-often includes vendor-specific information that is necessary to fully describe the manufacturer
-and model of the device including its security properties, the health
-of the device, and the level of confidence in the correctness of the information.
-Evidence is typically signed by the device (whether by hardware, firmware, or software on the
-device), and its appraisal in isolation would require Appraisal Policy to be based on
-device-specific details (e.g., a device public key).
+Evidence is a set of claims about the target device that reveal operational
+status, health, configuration or construction that have security relevance.
+Evidence is evaluated by a Verifier to establish its relevance, correctness, and timeliness.
+Claims need to be collected in a manner that is reliable.
+That way if the target device somehow can lie about them,
+the attesting environment is able to detect or circumvent it (lying).
+Evidence needs to be securely associated with the target device
+so that the Verifier cannot be tricked into accepting claims originating
+from a different device (that may be more trustworthy).
```

Comment:
```suggestion
from a different environment (that may be more trustworthy).
```

Reply:
My feeling is that the 'device' in this sentence refers to the 'Attester'.